core/src/test/java/io/questdb/griffin/engine/table/LatestByAllIndexedFilteredRecordCursorTest.java

```diff
@@ -0,0 +1,154 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2022 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.griffin.engine.table;
+
+import io.questdb.cairo.*;
+import io.questdb.cairo.security.AllowAllCairoSecurityContext;
+import io.questdb.cairo.sql.Function;
+import io.questdb.cairo.sql.RecordCursor;
+import io.questdb.griffin.AbstractGriffinTest;
+import io.questdb.griffin.engine.functions.columns.SymbolColumn;
+import io.questdb.griffin.engine.functions.constants.StrConstant;
+import io.questdb.griffin.engine.functions.eq.EqSymStrFunctionFactory;
+import io.questdb.std.IntList;
+import io.questdb.std.LongList;
+import io.questdb.std.ObjList;
+import io.questdb.test.tools.TestUtils;
+import org.junit.Ignore;
+import org.junit.Test;
+
+
+public class LatestByAllIndexedFilteredRecordCursorTest extends AbstractGriffinTest {
+    @Ignore("LatestByAllIndexedFilteredRecordCursorFactory applies filter after latest by is executed")
```

Comment:
Do you intend to leave it as ignored?

Reply:
The plan is to merge this PR and then move onto fixing the ignored tests by providing a factory that behaves applying the where clause first, then the latest by with the indexed column.